prod/php/ElasticOTel/Log/ElasticLogWriter.php

```diff
@@ -26,15 +26,32 @@
 use OpenTelemetry\API\Behavior\Internal\LogWriter\LogWriterInterface;
 use OpenTelemetry\API\Behavior\Internal\Logging;
 
-class ElasticLogWriter  implements LogWriterInterface
+class ElasticLogWriter implements LogWriterInterface
 {
-    public function write($level, string $message, array $context): void
+    /**
+     * @param array<mixed> $context
+     */
+    public function write(mixed $level, string $message, array $context): void
     {
-        \elastic_otel_log_feature(0, Level::getFromPsrLevel($level), LogFeature::OTEL, '', '', 0, $context['source'] ?? '', $message . ' context: ' . var_export($context, true));
```

Comment:
Do we have to have these calls stretched so far?

Reply:
Also is it the original file (on the left-hand side) while the updated file is on the other (right-hand) side and it's split inti multiple lines, isn't it?